modules/local/generate_images.nf

```diff
@@ -0,0 +1,33 @@
+process GENERATE_IMAGES {
+    tag "$meta.id"
+    label 'process_single'
+
+    if (params.enable_conda) {
+        exit 1, "Conda environments cannot be used when using the GENERATE_IMAGES module. Please use docker or singularity containers."
+    }
+    container 'genomehubs/blobtoolkit:4.0.7'
+
+    input:
+    tuple val(meta), path(blobdir)
+
+    output:
+    tuple val(meta), path("${blobdir}/cumulative.png") , emit: png
```

Comment:
Does this png file really have to be in blobdir ? Mixing outputs and inputs isn't best practice